src/features/user_profile/atoms/userProfile.ts

```diff
@@ -23,9 +26,24 @@ type profileResponse = UserProfileState;
 
 export const pageStatusAtom = createStringAtom<'init' | 'changed' | 'loading'>('init');
 
+// defaults, not provided by api/missing in profile
```

Comment:
comment is not clear for me, try
```suggestion
// default values provided for fields that were missing in the api call
```